src/test/scala/org/broadinstitute/dsde/workbench/sam/api/UserRoutesV1Spec.scala

```diff
@@ -61,7 +61,7 @@ class UserRoutesV1Spec extends UserRoutesSpecHelper{
       val res = responseAs[UserStatus]
       res.userInfo.userSubjectId.value.length shouldBe 21
       res.userInfo.userEmail shouldBe defaultUserEmail
-      res.enabled shouldBe Map("ldap" -> true, "allUsersGroup" -> true, "google" -> true, "tosAccepted" -> false)
+      res.enabled shouldBe Map("ldap" -> false, "allUsersGroup" -> true, "google" -> true, "tosAccepted" -> false)
```

Comment:
Right now, when we return a response map, we read from OpenDJ for the value we return in ldap, so this value is equivalent to (postgres.isEnabled && postgres.tosAccepted). I think it would be more correct to read/return the value from postgres, so we would return the postgres.isEnabled() value. Thoughts?

Reply:
I don't think I follow. I think you're talking about how the value for the ldap key is determined, but you might just be talking about this test.  I'm not sure :)

Reply:
That's an interesting question. I think there is still merit to returning the value that's actually in LDAP, for account debugging purposes, even if it should be derivable from other pieces of information. Could we still return `ldap` as-is and return `postgres.isEnabled()` as a new additional value?

Reply:
Definitely, I'll add a value called "isEnabled" to the result map to capture this. @sagehen03, the first one! I'm interested in how the value for the ldap key is determined, because it used to be true when the user was enabled, but now it's only true if the user is enabled & accepts ToS (when ToS is enabled of course!).